### ISSUE TITLE
Multilanguage relations do not save

### ISSUE THREAD

**Issue by CptMeatball** [OFFLINE-GmbH/oc-gdpr-plugin#21]
As relationship translations are still not working as of now, this might be a possible fix for the Cookiegroup -> Cookie relation translations.

This issue mentions a custom trait that overrides the multilanguage functionality for relationships. This might be a good temporary fix for ML relation fields.

https://github.com/rainlab/translate-plugin/issues/209

**Comment by ayumihamsaki** [comment]
I think could also add this code to the `Plugin.php` file to detect the RainLab Translation Plugin. Might be also useful...?

```
	public function registerMarkupTags()
    {
        // Check the translate plugin is installed
        $filters = array();
        if (!PluginManager::instance()->exists('RainLab.Translate')) {
            $filters['_'] = ['Lang', 'get'];
            $filters['__'] = ['Lang', 'choice'];
        }
    }
	

    /**
     * Check if a plugin exists and is enabled
     * @param plugin ID Namespace.Pluginname
     */
    public static function pluginExists($id)
    {
        $pi = PluginManager::instance();
        if (!$pi->findByIdentifier($id) || $pi->isDisabled($id)) {
            return false;
        }
        return true;
    }
```

**Comment by tobias-kuendig** [comment]
Thanks! This is fixed in 7c6e8f2.

**Comment by LukeTowers** [comment]
Fixed by https://github.com/octobercms/october/commit/13d710b161bdf65a19f38f974c259bc4997a162c
